lib/scanner/walk.go

```diff
@@ -197,7 +199,10 @@ func (w *walker) walk(ctx context.Context) chan protocol.FileInfo {
 }
 
 func (w *walker) walkAndHashFiles(ctx context.Context, fchan, dchan chan protocol.FileInfo) fs.WalkFunc {
+	var skip error // nil
```

Comment:
Why hoist this? It looks like it gets set to nil anyway at function entry, but it becomes part of the closure and I'm afraid I'm missing some cleverness?

Reply:
No cleverness, just a leftover from changing `handleItem` from closure to method.